js/device.js

```diff
@@ -99,6 +100,10 @@ class Device extends Backbone.Controller {
       : window.innerHeight || this.$window.height();
   }
 
+  setViewportHeight() {
+    document.documentElement.style.setProperty('--viewport-height', `${window.innerHeight}px`);
```

Comment:
Could you prepend this with `--adapt-viewport-height` as these are global to the browser.